modules/ducktests/tests/checks/utils/check_ignite_spec.py

```diff
@@ -79,19 +78,10 @@ def check_default_jvm_options__are_not_used__if_merge_with_default_is_false(serv
     assert len(spec.jvm_opts) == 0
 
 
-def check_boolean_options__go_after_default_ones_and_overwrite_them__if_passed_via_jvm_opt(service):
```

Comment:
Even if UnlockCommercialFeatures option is not used for JVM 11+  anymore it's not a reason to drop the test for the boolean options support.  Just choose another boolean option.  The -XX:+FlightRecorder would be ok I think.

Reply:
Returned the test, tweaked it to check FlightRecorder in options